Demo/Demo/ViewModels/CardVaultViewModel.swift

```diff
@@ -47,7 +47,7 @@ class CardVaultViewModel: VaultViewModel {
                     )
                 )
             } else if let vaultError {
-                if CardError.isThreeDSecureCanceled(vaultError) {
+                if let error = vaultError as? CoreSDKError, error == CardError.threeDSecureCanceledError {
```

Comment:
q: is it possible to make `vaultError` a `CoreSDKError?` to prevent needing a cast here?

Reply:
yes, good call